scripts/generate-docs-for-all-monitors.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# Copyright 2014-2020 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script which takes care of generating fragments of monitor documentation which can be generated
+# from the monitors code.
+#
+# It writes generated documentation files to docs/monitors/<monitor name>.md
+
+# Marker in the Markdown file which marks the line after which automatically generated content
+# should be appended
+AUTO_GENERATED_SECTION_MARKER="<!-- Auto generated content below. DO NOT edit manually, but run tox -egenerate-monitor-docs command instead -->"
+
+set +e
+
+# Verify systemd dependency is available
+python -c "import systemd" 2> /dev/null
```

Comment:
In the past, we've made an effort to make sure the `print_monitor_doc` script could be run for any monitor even if it's dependencies could not be loaded.  For example, we could print the documentation for the windows monitor even if we don't have the windows python libraries available.

I would suggest instead of jumping through hoops here to try to import systemd, we just fix the `journald` to try to import `systemd` but only fail when the monitor is actually attempted to be instantiated.

Reply:
Yeah, I was thinking about it - this should work as long as the actual auto-generated stuff doesn't reference any constants or similar from imported modules.

I will look into it and if that's indeed not the case, refactor the code so we import that module lazily.

It should also simplify the build config since we ca avoid installing that dependency, but we will still need to install it at some point anyway when we start adding more in depth and end to end tests for those monitors :)